stable/prometheus-operator/README.md

```diff
@@ -202,10 +202,10 @@ The following tables list the configurable parameters of the prometheus-operator
 | `prometheus.prometheusSpec.serviceMonitorSelector` | ServiceMonitors to be selected for target discovery. If {}, select all ServiceMonitors | `{}` |
 | `prometheus.prometheusSpec.serviceMonitorNamespaceSelector` | Namespaces to be selected for ServiceMonitor discovery. See [metav1.LabelSelector](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#labelselector-v1-meta) for usage | `{}` |
 | `prometheus.prometheusSpec.image.repository` | Base image to use for a Prometheus deployment. | `quay.io/prometheus/prometheus` |
-| `prometheus.prometheusSpec.image.tag` | Tag of Prometheus container image to be deployed. | `v2.7.1` |
+| `prometheus.prometheusSpec.image.tag` | Tag of Prometheus container image to be deployed. | `v2.8.1` |
```

Comment:
2.8.1 has some pretty broken remote write functionality, so I think it should be left as-is until that is stabalized

Reply:
@vsliouniaev Hi there! Could you please elaborate a bit on this topic or share a link. I would like to know more about this bug, since we are using 2.8.1 on production at the moment.

Reply:
The issue is still with the remote write feature - everything else appears to be working fine.
There are two open issues
https://github.com/prometheus/prometheus/issues/5424
https://github.com/prometheus/prometheus/issues/5389
with some fixes merged into the 2.9.0-rc.0 that went out yesterday: https://github.com/prometheus/prometheus/pull/5429
and some that did not: https://github.com/prometheus/prometheus/commit/1099999a79472d69f301fe42c40bb8493c62af64

We tested these fixes but it looks like there is still some instability:
https://github.com/prometheus/prometheus/issues/5451

The version 2.7.2 works fine with remote write, so updating to that should be fine, I think.

Reply:
@vsliouniaev I appreciate really your effort to provide sources of the issue. I will revert to `2.7.2`